Opio-Emmanuel-Omona/code-labs
# #159051218 Determine a color palette for the app

#### What does this PR do?
This PR determines a color palette for this app.

#### Description of Task to be completed?
- [x] Select an appropriate palette from materialpalette.com
- [x] Update the color.xml file

#### How should this be manually tested?
* clone this repo
```git clone https://github.com/Opio-Emmanuel-Omona/code-labs.git```
* check into this branch
```git checkout ft-color-palette-159051218```
* run the application through any prefered device.
Ensure that the application appears like that in the screenshot.

#### What are the relevant pivotal tracker stories?
[#159051218](https://www.pivotaltracker.com/story/show/159051218)

#### Screenshots
<img width="319" alt="Screen Shot 2019-03-20 at 2 55 38 PM" src="https://user-images.githubusercontent.com/26769886/54683106-de793480-4b21-11e9-8641-59f775c05a00.png">

##### Color Palettes
![palette](https://user-images.githubusercontent.com/26769886/54694702-b39ada80-4b39-11e9-93fc-c753efd4f172.png)
